core/modules/layout/plugins/relationships/user_from_node.inc

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * @file
+ * Class that holds information relating to the user from node relationship.
+ */
+class LayoutRelationshipUserFromNode extends LayoutRelationship {
```

Comment:
Change to `LayoutRelationshipAuthorFromNode` as per https://github.com/backdrop/backdrop-issues/issues/2134#issuecomment-754317337